source

```diff
@@ -89173,6 +89190,133 @@ interface <dfn interface>BeforeUnloadEvent</dfn> : <span>Event</span> {
+    is in a cross-origin iframe.</p></dd>
+    <dt><code data-x=""><var>notRestoredReasons</var>.<span subdfn data-x=
+    "dom-not-restored-reasons-src">src</span></code></dt>
+    <dd><p>Returns the <code data-x="attr-iframe-src">src</code> attribute of the document's
+    <span>node navigable</span>'s <span data-x="nav-container">container</span> if the document's
+    <span>node navigable</span>'s <span data-x="nav-container">container</span> is an
+    <code>iframe</code> element.<br>This can be null if not set or if the document is not in an
+    iframe.</p></dd>
+    <dt><code data-x=""><var>notRestoredReasons</var>.<span
+    subdfn data-x= "dom-not-restored-reasons-id">id</span></code></dt>
+    <dd><p>Returns the <code data-x="attr-id">id</code> attribute of the document's <span>node
+    navigable</span>'s <span data-x="nav-container">container</span> if the document's <span>node
+    navigable</span>'s <span data-x="nav-container">container</span> is an <code>iframe</code>
+    element.<br>This can be null if not set or if the document is not in an iframe..</p></dd>
+    <dt><code data-x=""><var>notRestoredReasons</var>.<span subdfn data-x=
+    "dom-not-restored-reasons-name">name</span></code></dt>
+    <dd><p>Returns the <code data-x="attr-iframe-name">name</code> attribute of the document's
+    <span>node navigable</span>'s <span data-x="nav-container">container</span> if the document's
+    <span>node navigable</span>'s <span data-x="nav-container">container</span> is an
+    <code>iframe</code> element.<br>This can be null if not set or if the document is not in an
+    iframe.</p></dd>
+    <dt><code data-x=""><var>notRestoredReasons</var>.<span subdfn data-x=
+    "dom-not-restored-reasons-url">url</span></code></dt>
+    <dd><p>Returns the document’s <span data-x="concept-document-url">url</span>. This is null if
+    the document is in a cross-origin iframe.<br>This is reported in addition to src, because the
+    url might have changed from the original src before navigating away.</p></dd>
+    <dt><code data-x=""><var>notRestoredReasons</var>.<span subdfn
+    data-x= "dom-not-restored-reasons-reasons">reasons</span></code></dt>
+    <dd><p>Returns an array of reasons that prevented the document from <a
+    href="#note-bfcache">being served from bfcache</a>.<br> Each reason is a string from the list
+    below.<br>This is null if the document is in a cross-origin iframe.</p></dd>
+    <dt><code data-x=""><var>notRestoredReasons</var>.<span subdfn
+    data-x= "dom-not-restored-reasons-children">children</span></code></dt>
+    <dd><p>Returns an array of <code>NotRestoredReasons</code> that are for the document’s
+    children.<br>This is null if the document is in a cross-origin iframe. </p></dd>
+  </dl>
+
+  <p>A <code>NotRestoredReasons</code> object has a <dfn
+    data-x="concept-not-restored-reasons-prevented-back-forward-cache">preventedBackForwardCache</dfn>,
```

Comment:
I think normally outside of the property getter we use normal names like "prevented bfcache state" or something like that, WDYT?

Reply:
Sounds good, I renamed this to "prevented bfcache state".